react/src/Deals.js

```diff
@@ -132,6 +133,23 @@ function StorageDealsContent(props) {
     </div>
 }
 
+function LegacyDealsLink(props) {
+    const {data} = useQuery(LegacyDealsCountQuery, {
+        pollInterval: 5000,
+        fetchPolicy: 'network-only',
+    })
+
+    if (!data || !data.legacyDealsCount) {
+        return null
```

Comment:
Since this call can take quite long on hosts with a lot of legacy deals, can we maybe still show it?
If there's no legacy deals, the page will be empty - but otherwise it can take a while to show up.

Reply:
Fair enough
https://github.com/filecoin-project/boost/pull/854#top